legal-api/src/legal_api/core/filing.py

```diff
@@ -364,7 +364,7 @@ def ledger(business_id: int,
             if (submitter := filing.filing_submitter) \
                 and submitter.username and jwt \
                     and not Filing.redact_submitter(filing.submitter_roles, jwt):
-                submitter_displayname = submitter.username
+                submitter_displayname = submitter.display_name or submitter.username
```

Comment:
@thorwolpert should we change the logic in `display_name` to return `username` for bcsc aswell instead of this?

Reply:
good question!
Was wondering about that as well but was not sure of the implementations of `display_name` and if that would an issue wherever else it's used.